vncviewer/vncviewer.man

```diff
@@ -205,9 +205,15 @@ Maximize viewer window.
 Start in full-screen mode.
 .
 .TP
-.B \-FullScreenAllMonitors
+.B \-FullScreenAllMonitors (DEPRECATED)
 Use all local monitors and not just the current one when switching to
-full-screen mode.
+full-screen mode. Replaced by \fB-FullScreenMode=all\fP
```

Comment:
If we can get rid of this from `--help` then it should also be removed from the man page IMO.

Reply:
Same here. We will keep it for now.